Update Kineto submodule

sraikund16 opened the pull request:
Differential Revision: D81087601

cyyever commented:
@sraikund16 To fix the error we could use `auto log_event = [=, this]() {` in CuptiActivityProfiler.cpp

sraikund16 commented:
> @sraikund16 To fix the error we could use `auto log_event = [=, this]() {` in CuptiActivityProfiler.cpp

I just changed it to explicitly capture instead of using `=`

sraikund16 commented:
@cyyever looks like it is failing again for the same linker issue. Do we need https://github.com/pytorch/kineto/pull/1131 to fix it?

sraikund16 commented:
@pytorchbot rebase

sraikund16 committed:
Update Kineto Hash (#161572)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/161572

Rollback Plan:

Differential Revision: D81087601

sraikund16 commented:
@pytorchbot merge

sraikund16 committed:
update kineto

sraikund16 commented:
@pytorchbot rebase
